src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/AspNetAppElasticBeanstalkLinux.csproj

```diff
@@ -45,7 +44,7 @@
   </ItemGroup>
   <!-- The project has run through the project template engine and "AWSDeployRecipesCDKCommonVersion" has been replaced with valid version number so a PackageReference is used. -->
   <ItemGroup Condition=" '$(RecipeCDKCommonVersion.ToUpper())' != 'AWSDEPLOYRECIPESCDKCOMMONVERSION' ">
-    <PackageReference Include="AWS.Deploy.Recipes.CDK.Common" Version="AWSDeployRecipesCDKCommonVersion"/>
+    <PackageReference Include="AWS.Deploy.Recipes.CDK.Common" Version="AWSDeployRecipesCDKCommonVersion" />
```

Comment:
nit: can you remove the whitespace change?

Reply:
This was done by VS doing its autoformatting. I think we should just take the change so we don't keep getting this change whenever somebodies VS does an autorformatting. It makes it consisted with other `PackageReferences` in the file.